### PR TITLE
Specify virtual env path when using a virtual env.

### PR DESCRIPTION
**Pull request by bastienwirtz**
In order to properly load the environment when using a virtual env, the `virtualenv` [parameter](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html#parameter-virtualenv) must be specified. 

This patch fix the the `externally-managed-environment` error ([since PEP 668 has been adopted recently](https://pythonspeed.com/articles/externally-managed-environment-pep-668/)): 
```ansible 
TASK [ansible-consul : Install netaddr dependency on controlling host (virtualenv)] *****************************************************************************************************************************************************************************************************
fatal: [consul-lab01 -> 127.0.0.1]: FAILED! => {"changed": false, "cmd": ["/usr/bin/python", "-m", "pip.__main__", "install", "netaddr"], "msg": "\n:stderr: error: externally-managed-environment\n\n× This environment is externally managed\n╰─> To install Python packages system-wide, try 'pacman -S\n    python-xyz', where xyz is the package you are trying to\n    install.\n    \n    If you wish to install a non-Arch-packaged Python package,\n    create a virtual environment using 'python -m venv path/to/venv'.\n    Then use path/to/venv/bin/python and path/to/venv/bin/pip.\n    \n    If you wish to install a non-Arch packaged Python application,\n    it may be easiest to use 'pipx install xyz', which will manage a\n    virtual environment for you. Make sure you have python-pipx\n    installed via pacman.\n\nnote: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.\nhint: See PEP 668 for the detailed specification.\n"}
```

Also, `is_virutalenv` was renamed to `virtualenv`, as it's not a boolean (could also be `virtualenv_path` and moved to the path section of the variable defaults?)

### CODE REVIEW

File: defaults/main.yml
Comment (nre-ableton) [inline defaults/main.yml]:
```suggestion
virtualenv: "{{ lookup('env','VIRTUAL_ENV') | default('') }}"
```

File: tasks/main.yml
Comment (nre-ableton) [inline tasks/main.yml]:
```suggestion
      when: virtualenv == ""
```
...since it will always be defined in the role defaults.

File: tasks/main.yml
Comment (nre-ableton) [inline tasks/main.yml]:
```suggestion
      when: virtualenv != ""
```

Again, it will always be defined in the role defaults.

**Review by nre-ableton — COMMENTED** [review]
Thanks for the PR. :+1: Sorry I missed this one while reviewing the others.

**Comment by bastienwirtz** [comment]
No worries @nre-ableton, thanks for the review !

**Review by nre-ableton — COMMENTED** [review]
LGTM, but can you please rebase and fix conflicts? You might need to fix some linting issues as well since we've disabled a bunch of ansible-lint suppressions.

**Comment by bastienwirtz** [comment]
Updated @nre-ableton

**Review by nre-ableton — APPROVED** [review]
LGTM, I'll just wait for CI to finish before merging.

**Pull request merged**
